resizable-polyline/demo.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Adds resizable geo polyline to map
+ *
+ * @param {H.Map} map                      A HERE Map instance within the application
+ */
+function createResizablePolyline(map) {
+  var svgCircle = '<svg width="20" height="20" version="1.1" xmlns="http://www.w3.org/2000/svg">' +
+      '<circle cx="10" cy="10" r="7" fill="transparent" stroke="red" stroke-width="4"/>' +
+      '</svg>',
+      polyline = new H.map.Polyline(
+        new H.geo.LineString([51.6, 21.8, 0, 52.3, 24, 0, 51.7, 25.8, 0, 50.6, 26, 0, 50, 24.3, 0, 49.2, 23.2, 0, 48, 23.2, 0]),
+        {
+          style: {fillColor: 'rgba(150, 100, 0, .8)', lineWidth: 10}
+        }
+      ),
+      polylineGroup = new H.map.Group({
+        volatility: true, // mark the group as volatile for smooth dragging of all it's objects
+        objects: [polyline]
+      }),
+      polylineTimeout;
+
+  // ensure that the polyline can receive drag events
+  polyline.draggable = true;
+
+  // create markers for each polyline's node which will be used for dragging
+  polyline.getGeometry().eachLatLngAlt(function(lat, lng, alt, index) {
+    var marker = new H.map.Marker(
+      {lat, lng},
+      {
+        icon: new H.map.Icon(svgCircle, {anchor: {x: 10, y: 10}})
+      }
+    );
+    marker.setVisibility(false);
+    marker.draggable = true;
+    marker.setData({'nodeIndex': index})
+    polylineGroup.addObject(marker);
+  });
+
+  // add group with polyline and it's nodes (markers)
+  map.addObject(polylineGroup);
+
+  // event listener for polyline group to show markers if moved in with mouse (or touched on touch devices)
+  polylineGroup.addEventListener('pointerenter', function(evt) {
+    var group = this;
+    if (polylineTimeout) {
+      clearTimeout(polylineTimeout);
+      polylineTimeout = null;
+    }
+
+    // show markers
+    group.forEach(function(object) {
```

Comment:
Why can't we have a second group and set visibility on it, that will show the user that groups are useful in many different cases? The same comes to the other examples.

Reply:
done